docs/spec/terminology.md

```diff
@@ -24,8 +24,8 @@
 | Public Key Commitment { #public-key-commitment } | The resulting [commitment](#commitment) obtained by applying the defined [commitment scheme](#operation-commitment-scheme) to a public key |
 | Recovery Commitment { #recovery-commitment }   | The resulting [commitment](#commitment) obtained by applying the defined [commitment scheme](#recovery-commitment-scheme) to the public key of a [recovery key pair](#recovery-key-pair)          |
 | Sidetree Node         | Executable code that implements all the required components, functionality, and rules specified in the Sidetree protocol specification.                            |
-| Transaction           | Ledger transaction that anchors a set of Sidetree operations, via a CAS URI for an associated Anchor File.          |
-| Anchor String  | The string anchored to the ledger, composed of the CAS URI to the [Anchor File](#anchor-file), prefixed with the declared operation count.                                               |
+| Transaction           | Ledger transaction that anchors a set of Sidetree operations, via a CAS URI for an associated Core Index File.          |
+| Core Index String  | The string anchored to the ledger, composed of the CAS URI to the [Core Index File](#core-index-file), prefixed with the declared operation count.                                               |
```

Comment:
Should anchor string also change the name? I thought that only file names will be changed. Was this name  chosen because the application is 'anchoring' this information on the blockchain or because it is pointing to anchor file?

Reply:
BTW, I am not asking for this to be changed - just making sure that it was not 'replace' mistake

Reply:
I agree with sandrask, I would want to keep the "anchor string" as is as it is more than just the core index.

Reply:
This was a replace error, good catch. Addressed in new commit.